product_docs/docs/biganimal/release/using_cluster/fault_injection_testing/index.mdx

```diff
@@ -97,11 +99,10 @@ To delete a write lead node from the cluster:
 Don't delete the Azure Kubernetes VMSS here or sub resources directly.
 !!!
 
-7. Browse to the Data Plane, choose Workloads, and locate the Kubernetes resources for your cluster to delete a chosen node.
+7. To delete a chosen node, browse to the data plane, select **Workloads**, and locate the Kubernetes resources for your cluster.
```

Comment:
I'm not sure if this edit is correct. I didn't see an action that would actually perform the delete. The edit is basically "goal before action,," but I'm not 100% sure that's what you meant.

Reply:
Thank you.  I have a draft copy of this to collaborate with the SDE on additional changes.  I'll add your edits into my draft before publishing.

Reply:
Okay, let me know when it's done and I'll cancel this pull request, assuming you meant to transfer all the edits.